app/models/manageiq/providers/redhat/infra_manager/ovirt_services.rb

```diff
@@ -1040,9 +1040,10 @@ def local_get_vnic_profiles_in_cluster(uid_ems_cluster)
       cluster_profiles
     end
 
-    def local_external_vnic_profiles
+    def local_external_vnic_profiles(uid_ems_cluster)
       external_profiles = {}
-      profiles = ext_management_system.external_distributed_virtual_lans
+      cluster = cluster_by_uid_ems(uid_ems_cluster)
+      profiles = cluster.parent_datacenter.external_distributed_virtual_lans
       external_networks = ext_management_system.external_distributed_virtual_switches
```

Comment:
@agrare, I think that also here we should get `external_distributed_virtual_switches` from the datacenter, right?